.github/workflows/lula.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+# This workflow runs a Lula scan against the PR to see if compliance has changed
+name: Lula Scan
+
+on:
+  pull_request:
+    branches: ["main"]
+    # milestoned is added here as a workaround for release-please not triggering PR workflows (PRs should be added to a milestone to trigger the workflow).
+    # labeled is added to support renovate-ready labelling on PRs
+    types: [milestoned, labeled, opened, reopened, synchronize]
+permissions:
+  contents: read
+
+jobs:
+  scan-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8
+      - name: Use Node.js 22
+        uses: actions/setup-node@a0853c24544627f65ddf259abe73b1d18a591444 # v5.0.0
+        with:
+          node-version: 22
+          cache: "npm"
+      - name: Run Lula Scan
+        run: |
+          npx --yes lula2 crawl
```

Comment:
We should probably pin the version here and then use renovate to update it? We do similar here - https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/tasks.yaml#L16-L17